manuscript/040_Three_Essential_Tools.md

```diff
@@ -154,28 +154,28 @@ Multiplication_ShouldResultInAMultiplicationOfTwoPassedNumbers()
+Phew, I hope I made the transition quite painless for you. Now the last thing to add -- as there is no `Main()` method anymore in the last example, you surely must wonder how we run those tests, right? Ok, the last big secret unveiled -- we use an external application for this (we will refer to it using the term **Test Runner**) -- we tell it which assemblies to run and it loads them, runs them, reports results etc. It can take various forms, e.g. it can be a console application, a GUI application or a plugin for an IDE. Here is an example of a stand-alone runner for the xUnit.NET framework:
 
-![XUnit.NET window](images/XUnit_NET_Window.png)
+![xUnit.NET window](images/XUnit_NET_Window.png)
 
 Mocking framework
 -----------------
 
-Mocking libraries automate runtime creation of objects (called “mocks") that adhere to specified interface. Aside from the creation itself, the libraries provide an API to configure our mocks on how they behave when certain methods are called on them and to let us inspect which calls they received.
+A mocking framework lets us create objects at runtime (called “mocks") that adhere to a certain interface. That interface is specified when the mock is created. Aside from the creation, the framework provides an API to configure the mocks on how they behave when certain methods are called on them and to let us inspect which calls they received.
 
-Mocking libraries are not as old as xUnit frameworks and were not present in TDD at very beginning. As you might be wondering why on earth do we need something like this, I will let you in on a secret: they were born with a goal of aiding a specific approach to object-oriented design in mind. This kind of design is what TDD can support quite well as you will soon experience.
+Mocking frameworks are not as old as test frameworks and they were not present in TDD at the very beginning. Why on earth do we need something like this? Well, they where invented to support a specific approach to object-oriented design that is a very good fit to TDD as you soon will experience.
 
-For now, however, I am trying to keep things simple. I will defer explaining the actual rationale for mocking libraries until later and instead give you a quick example where you can see them in action.
+For now, I'd like to keep things simple. I'll give you a quick example of a mocking framework in action and defer the explanation for their  rationale to a later chapter.
```

Comment:
Changed to: *I'll give you a quick example of a mocking framework in action now and defer further explanation for their rationale to a later chapter.*

I'll update the PR. Let's review the result again after merging. It'll be easier to spot issues from the updated text.